docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "MissingEscapingInFormatting",
+        "summary": "String literals passed to internal functions where not properly escaped.",
```

Comment:
Are we sure this is only about internal functions? It should concern ABIEncoderV2, too.

Reply:
Right, it only happens with the ABI encoder enabled